legacy/preferences/src/main/java/app/k9mail/legacy/preferences/GeneralSettingsManager.kt

```diff
@@ -16,4 +16,7 @@ interface GeneralSettingsManager {
     fun setMessageViewTheme(subTheme: SubTheme)
     fun setMessageComposeTheme(subTheme: SubTheme)
     fun setFixedMessageViewTheme(fixedMessageViewTheme: Boolean)
+
+    fun addListener(listener: GeneralSettingsChangeListener)
+    fun removeListener(listener: GeneralSettingsChangeListener)
```

Comment:
See above, could be part of a specialized class to handle listener registration.